test/lib/TestSetup.sol

```diff
@@ -50,18 +52,21 @@ abstract contract TestSetup {
     }
 
     TestMode internal testMode;
+    MirrorNode internal mirrorNode;
 
     function setUpMockStorageForNonFork() internal {
         if (HTS_ADDRESS.code.length == 0) {
             console.log("HTS code length is 0, non-fork test, code and data provided locally");
-            MirrorNodeMock mirrorNode = new MirrorNodeMock();
-            mirrorNode.deployHIP719Proxy(USDC, "USDC");
-            mirrorNode.deployHIP719Proxy(MFCT, "MFCT");
+            MirrorNodeMock mirrorNodeMock = new MirrorNodeMock();
+            mirrorNodeMock.deployHIP719Proxy(USDC, "USDC");
+            mirrorNodeMock.deployHIP719Proxy(MFCT, "MFCT");
+            mirrorNode = mirrorNodeMock;
```

Comment:
What is the intent of this change? What does it add?

Reply:
```solidity
    function test_HTS_getTokenInfo_should_revert_when_call_to_mirror_node_fails() external {
         address token = MFCT;
         vm.mockCallRevert(address(mirrorNode), abi.encode(mirrorNode.fetchTokenData.selector), abi.encode("mirror node error"));
         vm.expectRevert(bytes("getTokenInfo: failed to get token info"));
         HtsSystemContract(HTS_ADDRESS).getTokenInfo(token);
     }
```